Story_Builder/Tutorials_and_Examples/Building_Objects/BuildAnObjectFromScratchTutorial.md

```diff
@@ -0,0 +1,229 @@
+# Building an Object from Scratch
+
+For this tutorial, we will focus on the basics of creating an object from scratch. The process of creating any object can be broken down into three steps.
+
+1. Create the object
+2. Add details and descriptions
+3. Configure settings
+
+I will be building a wand in this example but feel free to create whatever type of object you want as you follow along. The completed object I am building can be found in `Tutorials:BuildingObjects:BuildAnObjectFromScratch:wand`.
+
+## Step 1: Create the object
+
+To create an object from scratch, use the +cobj command. The syntax for the command is as follows:
+
+```
++cobj syntax:
++cobj action         Tavern:coders:skott:myAction
++cobj helpnode       Tavern:coders:skott:myHelpnode
++cobj object         Tavern:coders:skott:myObject
++cobj propcontainer  Tavern:coders:skott:myPropcontainer
++cobj verb           Tavern:coders:skott:myVerb
+```
+
+This command is fairly straightforward and the only question you need to ask yourself is where do you want to save the object? The command will give you a suggestion but you can use anything you want. However, you should avoid using spaces or special characters, and do not save anything in the root directory. 
+
+`+cobj object wand` = saved to root directory = BAD
+
+`+cobj object Props:wand` = saved to "Props" folder = GOOD
```

Comment:
I'd suggest saying a little more here. Props:wand may be ok if you are on a stock server and you are the only one using it, but it might get confusing if lots of people are dropping props here. Props:unreviewed:wand is an option. But I personally suggest it go in a folder with your character name on it + props:unreviewed:wand. but once it is ready, and before it is live (i.e. other players can use it) it should be moved to the appropriate place.

Reply:
Good point :) Doc has been updated